organizations/default/client_assets/index.css

```diff
@@ -1 +1,166 @@
-/*  */
+body .input:focus {
+  border-color: #8b8b8b;
+}
+body .input.error {
+  border-color: #d93025;
+}
+body.input {
```

Comment:
Hi, @codesankalp Design looks good with some hover effects it can become better.
I would suggest some more improvements.
1. I think we can use fewer fillers like in the example shown below (use box shadows).
![image](https://user-images.githubusercontent.com/42765292/140615526-e28e1885-23e5-4dd4-bb7b-367c46267b36.png)

2. Its looks like everything is being highlighted with bold font. I suggest using fewer bold texts.

Reply:
Never mind these are only suggestions :smile: